documentation/TechnicalDesignDocument.tex

```diff
@@ -45,6 +45,25 @@
     %------------------------------------------------------------------------------------------------------------------------------------------------------- Requirements
     \section{Requirements}
     Functional and Non-functional requirements.
+    \subsection{Functional Requirements:}
+    \begin{itemize}
+        \item The program should support loading different Marama shapes using the Marama transpiler.
+        \item The program should let users add Marama objects to a world.
+        \item The program should let users see the Marama objects from any angle.
+        \item The program should allow users to attach Maramas to one another by their joints.
+        \item The program should check if attaching Maramas to one another is possible.
+        \item The program should support algebraic data structure Marama shapes.
+        \item The program should support polymorphic Marama shapes.
+        \item A game program should exist where the user can use the editor with predefined maps and limited Marama options.
+    \end{itemize}
+    \subsection{Non-Functional Requirements:}
+    \begin{itemize}
+        \item The logic of the program should be written entirely in scala.
+        \item The program should be made entirely open-source.
+        \item The program should be accessible on multiple platforms, mostly focusing on Android and PC.
+        \item A continuous integration system should be used to ensure that the program can properly build, even with multiple people working on it.
```

Comment:
> , even with multiple people working on it.
Do you not mean something along the lines of multiple platforms? I don't think continuous intergration is used to allow multiple people working at the same time.